src/scripts/displayLikes.js

```diff
@@ -0,0 +1,13 @@
+export default function displayLikes(response) {
+  const items = document.querySelectorAll('.item');
+  response.forEach((res) => {
+    items.forEach((item) => {
+      if (item.firstChild.nextSibling.innerHTML === res.item_id) {
+        const i = item.firstChild.nextSibling.nextSibling.firstChild.nextSibling.firstChild;
```

Comment:
[Optional]
You should name your variables better. Especially if you are daisy-chained with lots of selectors, it becomes hard to understand what you are selecting.